JitsiConference.js

```diff
@@ -3661,6 +3661,19 @@ JitsiConference.prototype.toggleE2EE = function(enabled) {
     this._e2eEncryption.setEnabled(enabled);
 };
 
+/**
+ * Sets the key for End-to-End encryption.
+ *
+ * @param keyInfo: {Object {
+                encryptionKey: CryptoKey
```

Comment:
Check how we document other inner options, usually it's a param line for each, with `keyInfo.XXX` as each item.